src/component/TodoList.tsx

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import { useContext } from 'react';
+import { TodoItem } from './TodoItem';
+import { TempTodo } from './TempTodo';
+import { TodosContext } from '../TodosProvider/TodosProvider';
+import { useLocation } from 'react-router-dom';
+type Props = {
+  isCompleted: boolean;
+};
+
+export const TodoList: React.FC<Props> = () => {
+  const { todos, filterTodos, tempTodo } = useContext(TodosContext);
+  const location = useLocation();
+  const filteredTodos = filterTodos(todos, location.hash);
+
+  return (
+    <section className="todoapp__main" data-cy="TodoList">
+      {/* This is a completed todo */}
+      {/* <div data-cy="Todo" className="todo completed">
+        <label className="todo__status-label">
+          <input
+            data-cy="TodoStatus"
+            type="checkbox"
+            className="todo__status"
+            checked
+          />
+        </label>
+
+        <span data-cy="TodoTitle" className="todo__title">
+          Completed Todo
+        </span> */}
+
+      {/* Remove button appears only on hover */}
+      {/* <button
+          type="button"
+          className="todo__remove"
+          data-cy="TodoDelete"
+        >
+          ×
+        </button> */}
+
+      {/* overlay will cover the todo while it is being deleted or updated */}
+      {/* <div data-cy="TodoLoader" className="modal overlay">
+          <div className="modal-background has-background-white-ter" />
+          <div className="loader" />
+        </div>
+      </div> */}
+
+      {/* This todo is an active todo */}
+      {filteredTodos.map(todo => (
+        <TodoItem key={todo.id} todo={todo} />
+      ))}
+      {tempTodo !== null && <TempTodo key={tempTodo.id} tempTodo={tempTodo} />}
```

Comment:
```suggestion
      {tempTodo !== null && <TodoItem todo={tempTodo} isLoading />}
```
temp todo is the same todo just with loader